4-storage/redshift-storage/sql/com.amazon.aws.cloudfront/wd_access_log_1.sql

```diff
@@ -1,57 +1,45 @@
-	ref_tree        varchar(1500) encode runlength not null,
-	ref_parent      varchar(255)  encode runlength not null,
-	-- Properties of this type
-	date_time           timestamp     encode raw not null,
-	x_edge_location     varchar(255)  encode text32k,
-	sc_bytes            integer        encode raw,
-	c_ip                varchar(45)   encode text32k,
-	cs_method           varchar(3)    encode runlength,
-	cs_host             varchar(2000) encode text32k,
-	cs_uri_stem         varchar(8192) encode text32k,
-	sc_status           varchar(255)  encode runlength,
-	cs_referer          varchar(8192) encode text32k,
-	cs_user_agent       varchar(2000) encode raw,
-	cs_uri_query        varchar(8292) encode text32k,
-	cs_cookie           varchar(1024) encode text32k,
-	x_edge_result_type  varchar(2000) encode text32k,
-	x_edge_request_type varchar(2000) encode text32k,
-	x_host_header       varchar(2000) encode text32k,
-	cs_protocol         varchar(5)    encode runlength,
-	cs_bytes            integer       encode raw,
-	time_taken          float         encode raw,
-	FOREIGN KEY(root_id) REFERENCES atomic.events(event_id)
+-- Source: iglu:com.amazon.aws.cloudfront/wd_access_log/jsonschema/1-0-4
+
+CREATE SCHEMA IF NOT EXISTS atomic;
+
+CREATE TABLE IF NOT EXISTS atomic.com_amazon_aws_cloudfront_wd_access_log_1 (
+    "schema_vendor"               VARCHAR(128)     ENCODE RUNLENGTH NOT NULL,
+    "schema_name"                 VARCHAR(128)     ENCODE RUNLENGTH NOT NULL,
+    "schema_format"               VARCHAR(128)     ENCODE RUNLENGTH NOT NULL,
+    "schema_version"              VARCHAR(128)     ENCODE RUNLENGTH NOT NULL,
+    "root_id"                     CHAR(36)         ENCODE RAW       NOT NULL,
+    "root_tstamp"                 TIMESTAMP        ENCODE RAW       NOT NULL,
+    "ref_root"                    VARCHAR(255)     ENCODE RUNLENGTH NOT NULL,
+    "ref_tree"                    VARCHAR(1500)    ENCODE RUNLENGTH NOT NULL,
+    "ref_parent"                  VARCHAR(255)     ENCODE RUNLENGTH NOT NULL,
+    "date_time"                   TIMESTAMP        ENCODE LZO       NOT NULL,
+    "x_edge_location"             VARCHAR(32)      ENCODE LZO,
+    "sc_bytes"                    DOUBLE PRECISION ENCODE RAW,
+    "c_ip"                        VARCHAR(45)      ENCODE LZO,
+    "cs_method"                   VARCHAR(3)       ENCODE LZO,
+    "cs_host"                     VARCHAR(2000)    ENCODE LZO,
+    "cs_uri_stem"                 VARCHAR(8192)    ENCODE LZO,
+    "sc_status"                   VARCHAR(3)       ENCODE LZO,
+    "cs_referer"                  VARCHAR(8192)    ENCODE LZO,
+    "cs_user_agent"               VARCHAR(2000)    ENCODE LZO,
+    "cs_uri_query"                VARCHAR(8192)    ENCODE LZO,
+    "cs_cookie"                   VARCHAR(4096)    ENCODE LZO,
+    "x_edge_result_type"          VARCHAR(32)      ENCODE LZO,
+    "x_edge_request_id"           VARCHAR(2000)    ENCODE LZO,
```

Comment:
@alexanderdean - in the previous version of this table, this field was mistakenly named "x_edge_request_type" rather than "x_edge_request_id". Does this mean some sort of migration script is needed?